client/package.json

```diff
@@ -23,7 +38,7 @@
     "react-dom": "^16.13.1",
     "react-markdown": "^5.0.3",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^4.0.3",
```

Comment:
What's up here? Please put back to 3.4.1

Reply:
ok